### PR TITLE
Added images to product cards

### PR DESCRIPTION
**Pull request by thumn**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
Product cards now have images of the actual products instead of the default robot! Images are loaded from the Products table in Airtable.

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## Relevant Links
N/A

### Online sources
N/A

[//]: # "Optional - copy links to any tutorial or documentation that was useful to you when working on this PR"

### Related PRs
N/A

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review
Log into the Clerk app as a clerk, and enter a customer's phone number to view the list of products. Since images are scaled down to 10% of their original size in Airtable, image loading should be fairly quick. Not all products have images as of now.

[//]: # "The order in which to review files and what to expect when testing locally"

## Next steps
- Populate the Airtable with missing product images.
- Restyle product cards to match Figma design.
- Stretch: Load product data and store in async storage when clerk first logs in (allows us to avoid having to reload images every time the clerk looks up a new customer).

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

## Tests Performed, Edge Cases
- Tested with one Airtable image URL at first for all products
- Tested with thumbnail images and various scales of original size (0.5, 0.3, 0.1)

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"

### Screenshots
image

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @anniero98 @wangannie

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

File: components/Product.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import React from 'react';
-import { Image, View } from 'react-native';
+import React from "react";
+import { Image, View } from "react-native";
```
Comment (anniero98) [inline components/Product.js]: I suspect our linter configs are conflicting 😂 I’ll check this later

File: components/Product.js [hunk 2]
```diff
@@ -10,9 +10,12 @@ import { ProductBody, ProductContainer } from '../styles.js';
 // TODO @tommypoa to use styled-components
 function Product({ product }) {
   return (
-    <View style={{ alignItems: 'center' }}>
+    <View style={{ alignItems: "center" }}>
       <Image
-        source={require('../assets/images/robot-dev.png')}
+        source={{
+          uri: product.image,
```
Comment (anniero98) [inline components/Product.js]: Does this mean that the images are hosted on Airtable? I’m just wondering if there are any drawbacks to this :o

Comment (thumn) [reply components/Product.js]: It's for the sake of convenience when HC updates or adds new products since everything belongs in the Products table :0 but the tradeoff is that there's a bit of a loading time. In the future it would be good to load everything in one go when the clerk first logs in!
